R/google-functions.R

```diff
@@ -94,7 +94,7 @@ dist_google <- function(from, to, google_api = Sys.getenv("GOOGLEDIST"),
   url = paste0(url_travel,
           google_api_param,
           google_api)
-  url = URLencode(url, repeated = FALSE, reserved = FALSE)
+  url = utils::URLencode(url, repeated = FALSE, reserved = FALSE)
```

Comment:
With this now add, we can (and should, I think) remove the previous import on line 27 `stplanr-package.R` I think.